presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -69,6 +71,8 @@
     private final ElasticsearchClient client;
     private final String schemaName;
 
+    private final Locale locale = Locale.getDefault();
```

Comment:
This will make the chosen local sensitive to what the JVM is configured to use. Let's use `Locale.ENGLISH` like we do everywhere else. 

Also, storing it in a field is unnecessary, so just inline it where it's used.

Reply:
Makes sense, I missed it, `ENGLISH` is used even in the same file :)